nengo_ocl/tests/test_simulator.py

```diff
@@ -5,7 +5,11 @@
 from nengo.builder import Model
 from nengo.builder.operator import DotInc, Reset
 from nengo.builder.signal import Signal
-from nengo.utils.testing import warns
+
+try:
+    from nengo.utils.testing import warns
+except ImportError:
+    from pytest import warns
```

Comment:
Can't we just always use `pytest.warns`? That's what I'd done in the nengo 2.7 compat branch.

Reply:
I agree, that this should be changed to what's done in #154. (This branch will give a merge conflict in that spot on rebase or merge.)

Reply:
Yeah I was just going for the minimal change, since I don't know what you're relying on elsewhere in the code.  But if `pytest.warns` works with earlier versions of your tests (which I would expect it to, but haven't explicitly confirmed 🤷‍♂️ ), then that'd make things simpler.